pkg/controller/config/config_controller.go

```diff
@@ -412,6 +424,14 @@ func (r *ReconcileConfig) validatePipeline(req reconcile.Request, cfg *op.Config
 		return reconcile.Result{}, err
 
 	}
+
+	labels := r.pipeline.Filter(mf.ByKind("Deployment"), mf.ByName(flag.PipelineControllerName)).Resources()[0].GetLabels()
+	version := labels["pipeline.tekton.dev/release"]
```

Comment:
same here, (let us move this into a function

Reply:
done